dist/mapd3.css

```diff
@@ -1,3 +1,8 @@
+body {
```

Comment:
Im wondering if dist/ folder shouldn't be excluded from the project (.gitignore) as this is the output being generated by the build script.

Reply:
Done